src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -135,8 +135,8 @@ def register_wl_container(cmd, client, vault_name, resource_group_name, workload
     param = ProtectionContainerResource(properties=properties)
 
     # Trigger register and wait for completion
-    result = client.register(vault_name, resource_group_name, fabric_name, container_name, param,
-                             cls=cust_help.get_pipeline_response)
+    result = client.begin_register(vault_name, resource_group_name, fabric_name, container_name, param,
+                                   cls=cust_help.get_pipeline_response, polling=False).result()
```

Comment:
May I ask why you need to disable the logic of polling in the code?

Reply:
The SDK was changed to return LROPoller instead, so I made the change to parse it correctly.

Reply:
Yes, what I mean is what problems would there be if using polling?

Reply:
You can try to obtain results from `LROPoller` in the following ways

https://github.com/Azure/azure-cli/blob/2caee1484074c595a836080aa4586b651b40380a/src/azure-cli/azure/cli/command_modules/network/custom.py#L4158

Reply:
> Yes, what I mean is what problems would there be if using polling?

I'm not entirely sure, honestly. I'm just following the pattern we've been using previously in our code:

https://github.com/Azure/azure-cli/blob/2caee1484074c595a836080aa4586b651b40380a/src/azure-cli/azure/cli/command_modules/backup/custom.py#L1403

Reply:
Okay, you can keep the original code style for now. If you have a need for the progress bar, you can consider changing the code style to the one we recommend in the future.